### PR TITLE
Change child titles to h3

### PR DESCRIPTION
**Pull request by tianhaopx**
Some titles are childs of high level titles. I changed their markdown header format from h2 `##` to h3 `###`.
Preview the result: [automatedtestingpython.md](http://rawgit.com/tianhaopx/tianhaopx.github.io/pytest-header/#!pages/automatedtestingpython.md), [automatedtestingcodeceptjs.md](http://rawgit.com/tianhaopx/tianhaopx.github.io/pytest-header/#!pages/automatedtestingcodeceptjs.md).

Reference: [PoC Result](https://docs.google.com/document/d/1FFpCPiTY3VmFtMHmYe78jOzpHLY_6ulhfdxJd6v7jXg/edit)

### CODE REVIEW

File: pages/automatedtestingcodeceptjs.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ See [Documentations on Google Docs](https://docs.google.com/document/d/1FFpCPiTY
 ## Motivation
 While writing tests with the Python `unittest` library is awesome, writing tests in CodeceptJS is super simple and much more intuitive.
 
-## Code
+### Code
```
Comment (mappuji) [inline pages/automatedtestingcodeceptjs.md]: Could you explain why you change all the Heading 2 in `automatedtestingcodeceptjs.md` except `Motivation` to Heading 3?

File: pages/automatedtestingpython.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ This, however, does not seem an ideal solution, since what works locally does no
 Perhaps, setting up individual accounts on Sauce Labs to perform testing on everyone’s personal repo before opening a pull request may be a better solution, in terms of development time.
 
 ## Windows - Installing Testing Development Environment
-## Installing Nation
+### Installing Nation
```
Comment (mappuji) [inline pages/automatedtestingpython.md]: I agree with this one

File: pages/automatedtestingpython.md [hunk 3]
```diff
@@ -41,7 +41,7 @@ Open git bash and type,
 
 Then, check in a browser if http://localhost:8081 is resolving to http://localhost:5981/apps/_design/bell/MyApp/index.html#login.
 
-## Installing Python, Pip, Selenium, Geckodriver, and Firefox
+### Installing Python, Pip, Selenium, Geckodriver, and Firefox
```
Comment (mappuji) [inline pages/automatedtestingpython.md]: I agree with this one

File: pages/automatedtestingpython.md [hunk 4]
```diff
@@ -71,10 +71,10 @@ Finally:
 13. `python test_login.py`
 
 ## MacOS/Ubuntu - Installing Testing Development Environment
-## Installing Nation
+### Installing Nation
```
Comment (mappuji) [inline pages/automatedtestingpython.md]: I agree with this one

File: pages/automatedtestingpython.md [hunk 5]
```diff
@@ -71,10 +71,10 @@ Finally:
...
 WIP - Needs some love.
 
-## Installing Python, Pip, Selenium, Geckodriver, and Firefox
+### Installing Python, Pip, Selenium, Geckodriver, and Firefox
```
Comment (mappuji) [inline pages/automatedtestingpython.md]: I agree with this one

File: pages/automatedtestingpython.md [hunk 6]
```diff
@@ -145,7 +145,7 @@ class ClassName(BaseCase): // don’t forget “BaseCase”
 The test functions are getting executed in alphabetical order.  
 For example, if you you have `def test_login_logout(self)` and `def test_incorrect_username(self)`, then `def test_incorrect_username(self)` will be executed first.
 
-## Test results explained
+### Test results explained
```
Comment (mappuji) [inline pages/automatedtestingpython.md]: I agree with this one

File: pages/automatedtestingpython.md [hunk 7]
```diff
@@ -160,18 +160,18 @@ It turns out that expected failure can be a confusing feature, so perhaps it’s
 
 ## Python Style Guidelines
 
-## File and Class names
+### File and Class names
```
Comment (mappuji) [inline pages/automatedtestingpython.md]: I agree with this one

File: pages/automatedtestingpython.md [hunk 8]
```diff
@@ -160,18 +160,18 @@ It turns out that expected failure can be a confusing feature, so perhaps it’s
...
 The name of each file should reflect the name of the class it contains, but it must always start by `test_` (i.e. if the class is `LoginTest`, the name of the file will be `test_login.py`).  
 This is particularly important for automatic test discovery!  
 Remember to add the `BaseCase` argument to every class (e.g., `LoginTest(BaseCase)`).
 
-## Test cases
+### Test cases
```
Comment (mappuji) [inline pages/automatedtestingpython.md]: I agree with this one

File: pages/automatedtestingpython.md [hunk 9]
```diff
@@ -160,18 +160,18 @@ It turns out that expected failure can be a confusing feature, so perhaps it’s
...
 Every test case should be named `test_` + name of test (e.g., `test_login`).  
 This is particularly important, because only methods that start with `test_`
 will be run.  
 Usually, the main method of each class should reflect the class and the file name (e.g., if you have a file named `test_login.py`, the class would be `LoginTest` and the main method should preferably be `test_login`).
 
-## Random rules
+### Random rules
```
Comment (mappuji) [inline pages/automatedtestingpython.md]: I agree with this one

File: pages/automatedtestingpython.md [hunk 10]
```diff
@@ -191,7 +191,7 @@ return x == 2
 
 WIP (Needs more love!)
 
-## QuantifiedCode
+### QuantifiedCode
```
Comment (mappuji) [inline pages/automatedtestingpython.md]: I agree with this one

File: pages/automatedtestingcodeceptjs.md (hunk 1 above)
Comment (tianhaopx) [inline pages/automatedtestingcodeceptjs.md]: No, I didn't change all h2. `Setting Up a Testing Development Environment` and `Current Challenges` are still h2. I changed them according to the title structures in this [google doc](https://docs.google.com/document/d/1FFpCPiTY3VmFtMHmYe78jOzpHLY_6ulhfdxJd6v7jXg/edit?usp=sharing).

Comment (dogi) [inline pages/automatedtestingcodeceptjs.md]: @mappuji @tianhaopx I can go either or ;)

Comment (tianhaopx) [inline pages/automatedtestingcodeceptjs.md]: @mappuji Does my explanation make sense?

Comment (mappuji) [inline pages/automatedtestingcodeceptjs.md]: @tianhaopx oh I see, you did it right.

**Pull request merged**